packages/zosjobs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS jobs SDK package will be documented in this file.
 
+## `7.2.1`
```

Comment:
TL;DR: change ```## `7.2.1` ``` to `## Recent Changes`

the error was complaining about the header.
we don't lock versions down in the PRs
instead, we use `## Recent Changes` (similar to Zowe Exlplorer's `## TBD Release`) 

And in our case, the automation will update the header once a version has been selected by the admins or the repo

Reply:
Done.